.buildkite/pipeline.yml

```diff
@@ -244,6 +244,12 @@
       python/ray/tests/test_output
     - bazel test --test_output=streamed --config=ci --test_env=RAY_MINIMAL=1 $(./scripts/bazel_export_options)
       python/ray/tests/test_runtime_env_ray_minimal
+    - bazel test --test_output=streamed --config=ci $(./scripts/bazel_export_options)
+      python/ray/tests/test_runtime_env
+    - bazel test --test_output=streamed --config=ci $(./scripts/bazel_export_options)
+      python/ray/tests/test_runtime_env_complicated
+    - bazel test --test_output=streamed --config=ci $(./scripts/bazel_export_options)
+      python/ray/tests/test_runtime_env_validation
```

Comment:
Why not move all runtime_env tests here instead of just these three?  They should all pass in this PR right?

Reply:
Can you help finding a list of them?

Reply:
Yup, here's a list, all in `/ray/tests`:
```
test_runtime_env.py
test_runtime_env_complicated.py
test_runtime_env_conda_and_pip.py
test_runtime_env_conda_and_pip_2.py
test_runtime_env_conda_and_pip_3.py
test_runtime_env_env_vars.py
test_runtime_env_fork_process.py
test_runtime_env_packaging.py
test_runtime_env_plugin.py
test_runtime_env_ray_minimal.py
test_runtime_env_validation.py
test_runtime_env_working_dir.py
test_runtime_env_working_dir_2.py
test_runtime_env_working_dir_3.py
test_runtime_env_working_dir_remote_uri.py
```

But after looking at it a bit more, I'm not sure if it's the right decision to include them all here.  It looks like the "Minimal install" test is only used as a kind of sanity check (it only runs some test_basic workloads).  If that's the case, then just running the three tests you added should be fine.

Reply:
I think the only concern we have is if any of those tests require extra dependencies. I believe the existing set of tests should cover this, but you can give me confirmation!

Reply:
Yup, existing tests are fine